src/algebra/ring/basic.lean

```diff
@@ -360,6 +386,10 @@ omit rα rβ rγ
 
 end ring_hom
 
+/-- A commutative semiring is a `semiring` with commutative multiplication. In other words, it is a
+type with the following structures: additive commutative monoid (`add_comm_monoid`), multiplicative
+commutative monoid (`comm_monoid`), distributive laws (`distrib`), and multiplication by zero law
+(`mul_zero_class`). -/
```

Comment:
Any reason you've used a full docstring for `comm_semiring` but a short one for `comm_ring`?

Reply:
Probably I've got more sleepy by the time I was writing the docstring for `comm_ring`.